src/components/ExpensiTextInput/index.native.js

```diff
@@ -8,7 +8,7 @@ const ExpensiTextInput = forwardRef((props, ref) => (
         // eslint-disable-next-line react/jsx-props-no-spreading
         {...props}
 
-        // Setting autoCompleteType to new-password throws an error on iOS, so fall back to password in that case
+        // Setting autoCompleteType to new-password throws an error on Android, so fall back to password in that case
```

Comment:
NAB: I see that this file changed from index.ios.js to index.native.js, so it is now used for IOS and Android, but the comment is only about Android throwing an error now. iOS doesn't throw this error?

If both throw this error, we should update the comment to say something like iOS/Android

Reply:
YUP 😁. I will update.